inc/save.php

```diff
@@ -42,16 +42,29 @@ function save() {
 
 	$sanitized = implode( PHP_EOL, $sanitized );
 
-	$postarr = array(
-		'ID'           => $post_id,
-		'post_title'   => 'Ads.txt',
-		'post_content' => $sanitized,
-		'post_type'    => 'adstxt',
-		'post_status'  => 'publish',
-		'meta_input'   => array(
-			'adstxt_errors' => $errors,
-		),
-	);
+	if ( 'app-adstxt' === $_post['adstxt_type'] ) {
+		$postarr = array(
```

Comment:
There's a lot of shared code between this and the block below. Looks like just the `post_title` and `post_type` are different, so can reduce the code down here to take advantage of that, similar to what we're doing in other places. Something like
```
$postarr = array(
	'ID'           => $post_id,
	'post_title'   => 'Ads.txt',
	'post_content' => $sanitized,
	'post_type'    => 'adstxt',
	'post_status'  => 'publish',
	'meta_input'   => array(
		'adstxt_errors' => $errors,
	),
);

if ( 'app-adstxt' === $_post['adstxt_type'] ) {
	$postarr = array(
		'post_title'   => 'App-ads.txt',
		'post_type'    => 'app-adstxt',
	);
}
```
Or could be more explicit and set the shared defaults and then `array_merge` the differences for both.